dev-packages/browser-integration-tests/suites/stacktraces/protocol_fn_identifiers/test.ts

```diff
@@ -5,7 +5,7 @@ import { sentryTest } from '../../../utils/fixtures';
 import { getFirstSentryEnvelopeRequest } from '../../../utils/helpers';
 
 sentryTest(
-  'should parse function identifiers that are protocol names correctly @firefox',
```

Comment:
this is here on purpose, that ensures this specific test is also run in FF :)

Reply:
Ah ok, thanks I'm reverting this 👍